lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -6916,6 +6968,16 @@ namespace Js
         return hr;
     }
 
+    HRESULT JavascriptLibrary::ProfilerRegisterBigInt()
+    {
+        HRESULT hr = S_OK;
+        REG_GLOBAL_CONSTRUCTOR(BigInt);
+
+        //DEFINE_OBJECT_NAME(BigInt);
```

Comment:
Why is this commented out? In the case bigint is disabled, should this function return E_FAIL?